scripts/azure_logs_integration.js

```diff
@@ -0,0 +1,699 @@
+var https = require("https");
```

Comment:
How much of this is diff from the DD file? Can you post a diff from that or better break this into two PRs first one is a checkin of original file as is and second one is the modifications.

We need to retain the license.

Can you get a review from @jt-kloudfuse or @vinayk-kfuse while I am out.

Reply:
Okay, its not very different; I have added some code to tweak the logs for us and modified the variables. I can post the original script in a separate PR. 
Regarding License, the original scripts says this on the top - 

`
// Unless explicitly stated otherwise all files in this repository are licensed
// under the Apache License Version 2.0.
// This product includes software developed at Datadog (https://www.datadoghq.com/).
// Copyright 2024 Datadog, Inc.`

Should I add this commented section?

Reply:
https://github.com/kloudfuse/customer/pull/14 - Original DD Script